src/Microsoft.ML.PipelineInference/PipelinePattern.cs

```diff
@@ -273,7 +273,7 @@ public PipelineResultRow ToResultRow()
         {
             var graphDef = ToEntryPointGraph();
 
-            return new PipelineResultRow($"{{'Nodes' : [{graphDef.Graph.ToJsonString()}]}}",
+            return new PipelineResultRow($"{{\"Nodes\" : [{graphDef.Graph.ToJsonString()}]}}",
```

Comment:
>\"Nodes\" [](start = 45, length = 9)

not huge fan of composing json strings manually, but I'm fine with this changes.